sdk/samples/iot/hub/src/paho_iot_hub_twin_example.c

```diff
@@ -483,7 +483,7 @@ static az_result build_reported_property(az_span* reported_property_payload)
       az_json_builder_init(&json_builder, AZ_SPAN_FROM_BUFFER(reported_property_buffer), NULL));
   AZ_RETURN_IF_FAILED(az_json_builder_append_begin_object(&json_builder));
   AZ_RETURN_IF_FAILED(az_json_builder_append_property_name(&json_builder, reported_property_name));
-  AZ_RETURN_IF_FAILED(az_json_builder_append_int32_number(&json_builder, reported_property_value));
+  AZ_RETURN_IF_FAILED(az_json_builder_append_int32(&json_builder, reported_property_value));
```

Comment:
FYI, for IoT folks, this is just a method rename, where we are dropping the `_number` suffix to make it shorter.